docs/umi/isoseq-bcstats.md

```diff
@@ -15,14 +15,17 @@ nav_order: 7
 4. Number of unique molecular barcodes matching this barcode
 5. Whether the barcode is Group/Cell barcode or a Molecular Barcode/UMI
 
-If `--json` is unset, JSON summary information is written to stderr ("/dev/stderr").
-Similarly, if '-o' is unset, output TSV information is written to stdout ("/dev/stdout").
+This tool should be run after `isoseq3 correct` to get the `rc` tag, which indicates which reads come from real cells (as opposed to ambient RNA).
```

Comment:
This should be run after correct but the rc tag is not necessary.

Reply:
OK MODIFIED